fe/fe-core/src/main/java/com/starrocks/persist/CreateCatalogLog.java

```diff
@@ -16,18 +16,25 @@ public class CreateCatalogLog implements Writable {
 
     @SerializedName(value = "catalogName")
     private String catalogName;
+    @SerializedName(value = "comment")
+    private String comment;
     @SerializedName(value = "properties")
     private Map<String, String> properties;
 
-    public CreateCatalogLog(String catalogName, Map<String, String> properties) {
+    public CreateCatalogLog(String catalogName, String comment, Map<String, String> properties) {
         this.catalogName = catalogName;
```

Comment:
What if I don't want to add any comments? will this catalog create with failure?

Reply:
> What if I don't want to add any comments? will this catalog create with failure?

MySQL [demodb]> CREATE EXTERNAL CATALOG hive_catalog_1 COMMENT "hive_catalog" PROPERTIES("type"="hive", "hive.metastore.uris"="thrift://127.0.0.1:9083");
Query OK, 0 rows affected (0.00 sec)

MySQL [demodb]> CREATE EXTERNAL CATALOG hive_catalog_2 PROPERTIES("type"="hive", "hive.metastore.uris"="thrift://127.0.0.1:9084");
Query OK, 0 rows affected (0.00 sec)

MySQL [demodb]> show catalogs;
+----------------+------+--------------+
| Catalog        | Type | Comment      |
+----------------+------+--------------+
| hive_catalog_2 | hive | NULL         |
| hive_catalog_1 | hive | hive_catalog |
+----------------+------+--------------+
2 rows in set (0.00 sec)

Reply:
> What if I don't want to add any comments? will this catalog create with failure?

If no comment is announced when executing create catalog ddl, comment would be NULL.It won't lead to failure.